.github/workflows/future-proof.yml

```
@@ -49,12 +47,10 @@ jobs:
         run: cargo update -v
 
       - name: Cargo test
-        uses: actions-rs/cargo@v1
         env:
           RUSTC_BOOTSTRAP: 1
-        with:
-          command: test
-          args: |
-            --features better-docs
-            ${{ matrix.cargo-locked }}
+        run: |
+          cargo test \
+            --features better-docs \
+            ${{ matrix.cargo-locked }} \
             ${{ matrix.feature-nightly-dropck_eyepatch }}
```

Can you add ` \` to this line as well (and then a `;` afterwards to keep the command well-formed)?

```suggestion
          cargo test \
            --features better-docs \
            ${{ matrix.cargo-locked }} \
            ${{ matrix.feature-nightly-dropck_eyepatch }} \
          ;
```

I really like having these things use trailing separators to improve the quality of subsequent `git diff`s 🙏